fury/actor.py

```diff
@@ -2395,10 +2397,17 @@ def vector_text(text='Origin', pos=(0, 0, 0), scale=(0.2, 0.2, 0.2),
         Text for the label.
     pos : (3,) array_like, optional
         Left down position of the label.
+    direction : (3,) array_like, optional, default: (0, 0, 1)
+        The direction of the label. If None, label will follow the camera.
     scale : (3,) array_like
         Changes the size of the label.
     color : (3,) array_like
         Label color as ``(r,g,b)`` tuple.
+    align_center : bool, default: True
+        If `True`, the anchor of the actor will be the center of the text.
+        If `False`, the anchor will be at the left bottom of the text.
+    extrusion: float, default: 0
+        Extrusion length of the text gives it a 3D look.
```

Comment:
`extrusion`?  it does not appear on the function arguments above

also, it is not used

Reply:
I was using it to specify extrusion but I thought making scale in Z axis do the job is a better choice

Reply:
@skoudoro  It's done